Chinese/Skinnyh/Skinnyh.md

```diff
@@ -0,0 +1,2 @@
+大白 [Click here](../Baymax/Baymax.md)
+Mayday [Click here](../mayDay/mayDay.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.